conf/pf.conf.defaults

```diff
@@ -757,6 +757,31 @@ pfcmd_warning_color=yellow
 #
 # Color of the success text for pfcmd
 pfcmd_success_color=green
+#
+# advanced.use_omapi_to_lookup_mac
+#
+# Use omapi to query dhcpd for the mac address of an ip
+use_omapi_to_lookup_mac=disabled
+#
+# advanced.omapi_key_name
+#
+# The omapi key name for signing messages
+omapi_key_name=
```

Comment:
What is my default value. Can I work without a default one ? If not, how to get it ?
So much questions